Makefile

```diff
@@ -54,11 +56,14 @@ debug: all
 
 -include $(DEPS)
 
-${OBJ} ${TEST_OBJ}: Makefile config.mk
+${OBJ} ${TEST_OBJ}: Makefile config.mk config.h
 
 %.o: %.c
 	${CC} -o $@ -c $< ${CFLAGS}
 
+config.h:
+	echo '#define SYSCONFFILE "${SYSCONFFILE}"' >> config.h
```

Comment:
It seems like my comment didn't come through, so here is it again.
This can be implemented without requiring a separate file by using DEFAULT_CFLAGS in config.mk. See how it's done for wayland enabling/disabling.

Reply:
I'll have look and will get back at you...